packages/react-server/package.json

```diff
@@ -24,50 +24,50 @@
   "dependencies": {
     "body-parser": "^1.15.2",
     "cls-q": "1.1.0",
-    "cookie": "0.1.2",
+    "cookie": "^0.3.1",
     "cookie-parser": "1.4.3",
-    "eventemitter3": "0.1.6",
-    "express-state": "1.2.0",
+    "eventemitter3": "^2.0.2",
+    "express-state": "^1.4.0",
     "lodash": "^4.16.4",
-    "mobile-detect": "1.3.0",
+    "mobile-detect": "^1.3.5",
     "q": "1.4.1",
     "querystring": "0.2.0",
-    "request-local-storage": "1.1.0",
-    "routr": "0.1.3",
-    "winston": "2.2.0"
+    "request-local-storage": "^1.1.1",
+    "routr": "^2.1.0",
```

Comment:
See #837 and #312.  The change had a few impacts:

1. They no longer do anything with the `navigate` option in https://github.com/redfin/react-server/blob/master/packages/react-server/core/context/Navigator.js#L53.  I have NO idea with the `type` variable is really referring to in our code...it seems super weird to me.  The Routr project said in their [commit message](https://github.com/yahoo/routr/pull/29): "If needed, the navigation logic should happen at a higher level or remove that special case."
2. According to #312, we have never been passing the HTTP method information to Routr anyway, so Routr never checked the method properties.
3. We are improperly encoding the page attribute `method` in the `routes_(client|server).js` file  because our documentation says it can be an array or string (Routr now supports an array), but we're encoding it as a string only.